### DISCUSSION TITLE
🥂 iLogtail开源社区线上Meetup第2期 8.25 19:30

### DISCUSSION THREAD

**Discussion by messixukejia** [alibaba/ilogtail#315] [Announcements]
# 概要
* 议题：iLogtail开源社区线上Meetup
* 时间：2022年8月25日19:30
* 直播地点： [B站直播间](https://live.bilibili.com/25786263)
* 参与人：[@messixukejia](https://github.com/messixukejia)、[@yyuuttaaoo](https://github.com/yyuuttaaoo)、[@EvanLjp](https://github.com/EvanLjp)、[@henryzhx8 ](https://github.com/henryzhx8)、[@tong549761863 ](https://github.com/tong549761863)
# 议题
* iLogtail社区版新版本1.1.1发布。 [@messixukejia](https://github.com/messixukejia)
* 采集配置热加载及K8s采集过滤能力支持。[@henryzhx8 ](https://github.com/henryzhx8)
* 基于eBPF 的iLogtail 无侵入监控。[@EvanLjp](https://github.com/EvanLjp)
* iLogtail社区证书首发。[@yyuuttaaoo](https://github.com/yyuuttaaoo)
* 阿里云Serverless应用引擎日志采集架构案例分享。[@tong549761863 ](https://github.com/tong549761863)

<img src="https://user-images.githubusercontent.com/13250519/186591990-1a86f077-b213-4926-bdd9-528580a7e212.png" height="50%" width="50%" >

**Comment by yyuuttaaoo** [comment]
视频回放 https://www.bilibili.com/video/BV1ht4y1M7VX/?spm_id_from=333.999.0.0
